homework/unique_ptr/UniquePointer.hpp

```diff
@@ -29,27 +31,27 @@ class UniquePointer {
         return *this;
     }
 
-    const pointer get() const noexcept {
+    PointerToConst get() const noexcept {
         return pointer_;
     }
 
-    pointer get() noexcept {
+    Pointer get() noexcept {
         return pointer_;
     }
 
-    void reset(pointer ptr = pointer()) noexcept {
+    void reset(Pointer ptr = Pointer()) noexcept {
         delete pointer_;
         pointer_ = ptr;
     }
 
     const UniquePointer& operator*() const noexcept {
-        return *get();
+        return *pointer_;
     }
 
     UniquePointer& operator*() noexcept {
```

Comment:
Return element_type&.

Reply:
Ohh, don't know why haven't I noticed this. Kind of a blackout on my side ;)